### PR TITLE
Sequestered cloner Z5 prefab

### PR DESCRIPTION
**Pull request by MarkNais**
[ENHACEMENT][MAPPING]
## About the PR 
Adds the sequestered cloner prefab to the asteroid Z-level generation (25% chance).
Has lore about Syndicate sleeper agents, a TEG puzzle, and some CE-gear + omnizine as a reward.
APC code slightly refactored to allow overriding of a portion of code. Added a "puzzle APC".

I could make a trench variant, but neither ocean map use a TEG (which is featured in the prefab).
Relies on #871 being merged.

## Why's this needed? 
More Z5 prefabs, more good.

## Changelog

```
(u)MarkNstein:
(+)New Z5 asteroid field prefab: Sequestered Cloner. Contains lore, a puzzle, and loots!
```

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -1270,10 +1249,36 @@ var/zapLimiter = 0
 
 	src.updateDialog()
 
+// set channels depending on how much charge we have left
+/obj/machinery/power/apc/proc/check_channle_thresholds()
```
Comment (glassofmilk) [inline code/modules/power/apc.dm:1253]: Spelling - I think this should be check_channel_thresholds()

Comment (MarkNais) [reply code/modules/power/apc.dm]: Aye! thanks for catching that!

**Comment by ZeWaka** [comment]
This PR is unclean, you seem to add ID dispensers in it.

**Comment by MarkNais** [comment]
@ZeWaka 
Fixed. 
Still new to Git, Was hoping the merge of the precursor PR would "resolve" that unclean file. But I better understand now how the history and such works.

**Pull request merged**